quic/s2n-quic-core/src/recovery/bbr.rs

```diff
@@ -461,7 +485,10 @@ impl BbrCongestionController {
         //#     inflight += 2
         //#   return inflight
 
-        let offload_budget = 1; // TODO: Track offload budget
+        //= https://tools.ietf.org/id/draft-cardwell-iccrg-bbr-congestion-control-02#4.5.4
+        //# BBRUpdateOffloadBudget():
```

Comment:
I don't think `//#` has any special meaning in Rust, what you are seeing here is the "Duvet" compliance feature we've built that allows us to cite any IETF-style specification in the code, and then build a report that shows what requirements from the RFC is implemented and where.

If you click the **`compliance / report`** [Details link](https://dnglbrstg7yg.cloudfront.net/6153884a038f106d57d67a287476e60196f4a002/compliance.html#/) on the bottom of this PR, you'll see the report that was built for this PR.

Reply:
Here's a direct link to the section of the BBRv2 RFC where this citation shows up: https://dnglbrstg7yg.cloudfront.net/6153884a038f106d57d67a287476e60196f4a002/compliance.html#/spec/https%3A%2F%2Ftools.ietf.org%2Fid%2Fdraft-cardwell-iccrg-bbr-congestion-control-02/section-4.5.4

Reply:
Yep. The BBRv2 specification happens to have a ton of pseudocode, and we've found it helpful to include a citation of that pseudocode in our implementation to show that we have implemented it correctly. The compliance tool doesn't just have to cite pseudocode, it can cite any requirement (really any sentence) in the RFC. Sentences containing certain keywords like `MUST` or `SHOULD` are treated specially in the report.